### DISCUSSION TITLE
Welcome to TerminatorZ Discussions!

### DISCUSSION THREAD

**Discussion by blackhatethicalhacking** [blackhatethicalhacking/TerminatorZ#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Amesleh** [comment]
Hi, how do we fix this?
Usage: httpx [OPTIONS] URL

Error: No such option: -e

**Comment by blackhatethicalhacking** [comment]
Hi, First, make sure when you test :

`wayback example.com | httpx -verbose > test.txt`

works without issues, if not and you are getting this, please install httpx and Wayback based on the dependencies and try again TerminatorZ.

refer to this as well:

https://github.com/blackhatethicalhacking/TerminatorZ/issues/4

I explain how to handle it.
